test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XplatListPackageJsonRendererTests.cs

```diff
@@ -0,0 +1,1249 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.CommandLine.XPlat;
+using NuGet.CommandLine.XPlat.Utility;
+using NuGet.Test.Utility;
+using Xunit;
+using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using NuGet.Configuration.Test;
+using NuGet.Configuration;
+using NuGet.Common;
+using System.Threading;
+using NuGet.Protocol;
+using System.Linq;
+using NuGet.Versioning;
+using System;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [Collection("NuGet XPlat Test Collection")]
+    public class XplatListPackageJsonRendererTests
+    {
+        [Fact]
+        public void JsonRenderer_ListPackage_SucceedsAsync()
+        {
+            // Arrange
+            var reportType = ReportType.Default;
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                string consoleOutputFileName = Path.Combine(pathContext.SolutionRoot, "consoleOutput.txt");
+                string frameWork5 = "net5.0";
+                string frameWork31 = "netcoreapp3.1";
+                var projectAPath = Path.Combine(pathContext.SolutionRoot, "projectA.csproj");
+                var projectBPath = Path.Combine(pathContext.SolutionRoot, "projectB.csproj");
+
+                using (FileStream stream = new FileStream(consoleOutputFileName, FileMode.Create))
```

Comment:
Json output is redirected to `consoleOutput.txt` file. Later this file content compared to expected output for assertion.